org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -249,12 +249,16 @@ class Translator {
         return this.postDeviceResource(deviceId, "colourChroma", payload);
     }
 
-    postSubscribe(callbackUrl, verificationRequest) {
-        return winkHub._subscribe(deviceType, deviceId, callbackUrl, verificationRequest);
+    postSubscribe(subscriptionInfo) {
```

Comment:
This is supported in the hubController (and in the hubController tests).  I'll add to the CLI.

---
In reply to: [92025415](https://github.com/openT2T/translators/pull/136#discussion_r92025415) [](ancestors = 92025415)